modin/_compat/pandas_api/latest/dataframe.py

```diff
@@ -68,6 +68,67 @@ def apply(
             func, axis=axis, raw=raw, result_type=result_type, args=args, **kwargs
         )
 
+    def compare(
+        self,
+        other,
+        align_axis=1,
+        keep_shape=False,
+        keep_equal=False,
+        result_names=("self", "other"),
```

Comment:
```suggestion
        align_axis: Union[str, int] = 1,
        keep_shape: bool = False,
        keep_equal: bool = False,
        result_names: tuple = ("self", "other"),
```

Reply:
right, I'm going to copy the hints from [pandas source](https://github.com/pandas-dev/pandas/blob/v1.5.0/pandas/core/frame.py#L7720-L7848)